stream-chat-android-sample/src/main/kotlin/io/getstream/chat/sample/feature/user_login/UserLoginFragment.kt

```diff
@@ -77,6 +78,17 @@ class UserLoginFragment : Fragment() {
                 viewModel.targetChannelDataReceived(cid)
             }
         }
+
+        activity?.apply {
```

Comment:
Nit: we could use [`requireActivity`](https://developer.android.com/reference/kotlin/androidx/fragment/app/Fragment#requireActivity()) here in a few places